components/mailchimp/sources/new-order/new-order.js

```diff
@@ -0,0 +1,115 @@
+const common = require("../common-webhook");
+const { mailchimp } = common.props;
+
+module.exports = {
+  ...common,
+  key: "mailchimp-new-order",
+  name: "New Order",
+  description:
+    "Emit an event when a new order is added to your store, or Mailchimp account.",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    mailchimp,
+    server: { propDefinition: [mailchimp, "server"] },
+    storeId: {
+      type: "string",
+      label: "Store Id",
+      description:
+        "The unique ID of the store you'd like to watch for new orders. Leave empty to watch for orders on your Mailchimp account.",
+      optional: true,
+    },
+    campaignId: {
+      type: "string",
+      label: "Campaign Id",
+      description:
+        "Watch only for new orders with a specific Campaign Id value.",
+      optional: true,
+    },
```

Comment:
These should be obtained through an API call, not expect the user to look for them.
The same with `customerId`.

Reply:
This has been previously been addressed.